src/woo_publications/publications/models.py

```diff
@@ -133,6 +138,43 @@ def get_owner(self) -> ActingUser | None:
         assert isinstance(log, TimelineLogProxy)
         return log.acting_user[0]
 
+    def revoke_own_published_documents(
+        self, user: User | ApiUser, remarks: str | None = None
+    ) -> None:
+        documents = (
```

Comment:
```suggestion
        published_documents = (
```
can't hurt to be more explicit when working with this variable